src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -34,7 +34,8 @@ foam.CLASS({
     { name: 'ERROR_MSG', message: 'There was a problem creating your account.' },
     { name: 'SELECTION_TEXT', message: 'Select your country' },
     { name: 'SELECTION', message: 'Please select...' },
-    { name: 'VALIDATION_ERR_TEXT', message: 'Please enter job title'}
+    { name: 'VALIDATION_ERR_TEXT', message: 'Please enter job title'},
```

Comment:
SELECTION_TEXT, SELECTION, AND VALIDATION_ERR_TEXT  not used

Reply:
I don't think that validationPredicates has a reference to 'this', so this.VALIDATION_ERR_TEXT would resolve to undefined. I double checked the file system, and all values for errorMessage are hardcoded strings. I tried setting the errorMessage to 'VALIDATION_ERR_TEXT' like it was done in the previous revision, but get this...
<img width="407" alt="Screen Shot 2020-07-08 at 11 04 08 AM" src="https://user-images.githubusercontent.com/57911258/86935393-c8262c00-c10a-11ea-90f9-23be14f5f4aa.png">

Reply:
hmmm it was working before your change to a errorString - doesn't make sense that if you change the message.message it should stop working. I'd suggest checking the spelling of the variables - or setting a debug.